helm-charts/Makefile

```diff
@@ -2,7 +2,7 @@ PROJECT_NAME=helm-charts
 
 RELEASE_VERSION ?= latest
 CHART_SEMANTIC_RELEASE_VERSION ?= $(shell cat ../release.version | tr A-Z a-z)
-CHART_NAME=strimzi-kafka-operator
+CHART_NAME=strimzi-cluster-operator
```

Comment:
This whole chart renaming think shouldn't be here, as it never got into master.

Reply:
I am not sure how is this possible, but I will remove the commit and all changes.